grant-project/grant-monorepo
# Comment, Placeholder & Routes to less.

Part of #41 

### What this does
- Converts Comment, Placeholder & Routes styled-component styles to less.

### Test
- ensure placeholder looks good:
![screen shot 2018-09-20 at 9 13 46 pm](https://user-images.githubusercontent.com/11430954/45856935-2dcddc00-bd1c-11e8-9f7e-a54cea93aa45.png)
- Routes/global html.less - adjust screen width and observe that header links become smaller when < 900px:
![untitled](https://user-images.githubusercontent.com/11430954/45857008-76859500-bd1c-11e8-8f1e-977c6cfbf8b3.gif)
- ensure comments render properly:
![screen shot 2018-09-20 at 9 13 24 pm](https://user-images.githubusercontent.com/11430954/45857018-800efd00-bd1c-11e8-9e8a-7cba053ebd20.png)
